asreviewcontrib/insights/entrypoint.py

```diff
@@ -80,11 +80,17 @@ def execute(self, argv):
         )
         args = parser.parse_args(argv)
 
+        asreview_files = [
+            str(file)
```

Comment:
Personally I always get a bit confused when reading list comprehensions with many parts. I really have to think hard to make sense of it. I would prefer something like
```[python]
filepaths = []
for path in args.asreview_files:
    path = Path(path)
    ...
    filepaths.append(some_filepath) 
```